st2api/Makefile

```diff
@@ -59,7 +59,7 @@ all: install
 
 install: wheelhouse changelog
 
-post_install:
+post_install: bdist_wheel
```

Comment:
Actually it seem to work good with the latest pip which we use, i.e. I haven't installed `wheel` package anywhere explicitly.
```
ls -la /tmp/wheelhouse | grep -c wheel
0
```
Yeah, It's not needed now.